Assets/Tile walking mini game/Scripts/PlayerControls.cs

```diff
@@ -89,8 +88,12 @@ private void MovePlayerToTile(int Xmovement, int Ymovement)
         }  
     }
 
-    private void DisablePlayerMovement()
+    private void DisablePlayerMovement(GameState gameState)
     {
+        if(gameState.CheckCurrentState("lose") ||

```

Comment:
Since you do this same OR check in another place `Timer.cs`, maybe you could have a method for it.